### PR TITLE
Refactored JJDispute's PK to a Long.

### PR DESCRIPTION
**Pull request by KDNeufeld**
# Description

This PR includes the following proposed change(s):

- Refactored JJDispute to use a new PK of type Long rather than the String ticketNumber.
- Removed unused class DisputantContactInformation
- Updated tests

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

mvn test
OracleAPI Swagger successfully returns all JJDispute records initialized by data.sql

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by pbolduc — COMMENTED** [review]
Why do we overload the meaning of "Ticket Number". If this is an "I'd", then I'd or ticketId is better.

**Comment by KDNeufeld** [comment]
> Why do we overload the meaning of "Ticket Number". If this is an "id", then is or ticketId is better. (Darn auto correct)

The JJDispute model had a String TicketNumber as a PK where the data was the ticket number, ie. "EA07789631". (called invoice_number in the test-ticket-data.csv file you created)

I created a new PK for JJDispute to be a simple Long instead of using this string. This more closely matches the data coming back from ORDS.

**Review by lfdgov — APPROVED** [review]

File: src/backend/oracle-data-api/src/main/resources/data.sql [hunk 1]
```diff
@@ -1,127 +1,96 @@
...
-(25, 'AO38375806', 'System',   LOCALTIMESTAMP, 'System',    LOCALTIMESTAMP,  '1298 Mapleview Place',  '1976-03-15', '6455895',              'colm.ohiggins@nttdata.com', 'Charles',         'BC',     'Laroque'),
-(26, 'AU46117416', 'System',   LOCALTIMESTAMP, 'System',    LOCALTIMESTAMP,  '1298 Mapleview Place',  '1976-03-15', '6455895',              'colm.ohiggins@nttdata.com', 'Rebecca',         'BC',     'Klein'),
-(27, 'ST04605408', 'System',   LOCALTIMESTAMP, 'System',    LOCALTIMESTAMP,  '1298 Mapleview Place',  '1976-03-15', '6455895',              'colm.ohiggins@nttdata.com', 'Brian',           'BC',     'McIntosh');
-	
+(ID, TICKET_NUMBER, CREATED_BY, CREATED_TS,     MODIFIED_BY, MODIFIED_TS,    COURTHOUSE_LOCATION,              GIVEN_NAMES,       SURNAME,     OFFENCE_LOCATION, FINE_REDUCTION_REASON,    TIME_TO_PAY_REASON,               ENFORCEMENT_OFFICER, JJ_ASSIGNED_TO, POLICE_DETACHMENT, VIOLATION_DATE,     STATUS, SUBMITTED_TS,   ICBC_RECEIVED_DATE, HEARING_TYPE, LAW_FIRM_NAME,                  LAWYER_SURNAME, LAWYER_GIVEN_NAME1, LAWYER_GIVEN_NAME2, LAWYER_GIVEN_NAME3, INTERPRETER_LANGUAGE_CD, WITNESS_NO, DISPUTANT_ATTENDANCE_TYPE) VALUES
```
[the remark this answers is not in the document]
Comment (KDNeufeld) [reply src/backend/oracle-data-api/src/main/resources/data.sql]: Good catch. Incorporated into the JJDispute table.

**Comment by pbolduc** [comment]
The reason it is called invoice number in the CSV is because that is the name of the field returned by RSI. I wanted to ensure the processing logic on ticket search was the exact same code regardless of calling RSI service or read is ng our mock CSV data.

**Comment by pbolduc** [comment]
What I am trying to promote is "ubiquitous language" for our project.

**Pull request merged**